Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:carbon-alpine
+
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app/
+RUN yarn install
+RUN yarn build
+
+ENV NODE_ENV="production"
+EXPOSE 3000
+
+CMD ["yarn", "run", "start"]
```

Comment:
Does `CMD` has to run as `root` ?

Reply:
I don't think it should pose a problem to run it as the default `node` user (see: https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#non-root-user). This should either be specified in the `Dockerfile`, but it can also be done directly when running the machine with `-u node`